django_countries/fields.py

```diff
@@ -15,13 +16,16 @@
 from django_countries.conf import settings
 
 _entry_points: Iterable[Any]
-try:
+if sys.version_info >= (3, 8):
     import importlib.metadata
 
-    _entry_points = importlib.metadata.entry_points().get(
-        "django_countries.Country", []
-    )
-except ImportError:  # Python <3.8
+    if sys.version_info >= (3, 10):
+        _entry_points = importlib.metadata.entry_points(group="django_countries.Country")
+    else:
+        _entry_points = importlib.metadata.entry_points().get(
+            "django_countries.Country", []
+        )
+else:
     import pkg_resources
```

Comment:
pkg_resources is available because of setuptools. But you can create virtualenv without setuptools, so pkg_resources should be added as a dependency for <=3.8 or use `importlib-metadata` backport

Reply:
Using the backport would be best, it can be declared conditionally in `install_requires` in `setup.cfg` with:

```
importlib-metadata >= 3.6.0 ; python_version < "3.10"
```

Then we could do:

```py
if sys.version_info >= (3, 10):
    from importlib.metadata import entry_points
else:
    from importlib_metadata import entry_points
_entry_points = entry_points(group="django_countries.Country")
```

Reply:
Oh, so the backport version 3.6.0 and above is offering the new API from Python 3.10 and above? That's neat. Will try to make the change when I find some time

Reply:
It should, yes: https://importlib-metadata.readthedocs.io/en/latest/history.html#v3-6-0